src/applications/disability-benefits/4142/config/form.js

```diff
@@ -0,0 +1,252 @@
+// _ from 'lodash/fp';
+
+// Example of an imported schema:
+// import fullSchema from '../22-4142-schema.json';
+// In a real app this would be imported from `vets-json-schema`:
+// import fullSchema from 'vets-json-schema/dist/22-4142-schema.json';
+
+// In a real app this would not be imported directly; instead the schema you
+// imported above would import and use these common definitions:
+import commonDefinitions from 'vets-json-schema/dist/definitions.json';
+import PrivateProviderTreatmentView from '../../../../platform/forms/components/ServicePeriodView';
+import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+
+// const { } = fullSchema.properties;
+
+// const { } = fullSchema.definitions;
+
+import {
+  medicalRecDescription,
+  letUsKnow,
+  aboutPrivateMedicalRecs,
+  limitedConsentDescription,
+  summary,
+} from '../helpers';
+
+
+import PhoneNumberWidget from 'us-forms-system/lib/js/widgets/PhoneNumberWidget';
+
+const { fullName, ssn, date, dateRange, usaPhone } = commonDefinitions;
+
+// Define all the fields in the form to aid reuse
+const formFields = {
+  fullName: 'fullName',
+  ssn: 'ssn',
+  toursOfDuty: 'toursOfDuty',
+  privateMedicalProvider: 'privateMedicalProvider',
+  viewNoDirectDeposit: 'view:noDirectDeposit',
+  viewStopWarning: 'view:stopWarning',
+  bankAccount: 'bankAccount',
+  accountType: 'accountType',
+  accountNumber: 'accountNumber',
+  routingNumber: 'routingNumber',
+  address: 'address',
+  email: 'email',
+  altEmail: 'altEmail',
+  phoneNumber: 'phoneNumber',
+};
+
+// Define all the form pages to help ensure uniqueness across all form chapters
+const formPages = {
+  applicantInfo: 'applicantInfo',
+  uploadInfo: 'uploadInfo',
+  serviceHistory: 'serviceHistory',
+  treatmentHistory: 'treatmentHistory',
+  contactInfo: 'contactInfo',
+  directDeposit: 'directDeposit',
+};
+
+const formConfig = {
+  urlPrefix: '/',
+  submitUrl: '/v0/private_medical_record_auth/submit',
+  trackingPrefix: 'complex-form-',
```

Comment:
:smirk:

Reply:
Hey! I'm not sure what your comment means. Is there something I need to change? Thanks!

Reply:
@imeecuison This tracking prefix should be updated or removed; `complex-form-` came from the form generator

Reply:
Heh, sorry. I didn't realize this came from the generator! :laughing:

It probably makes sense for this to be the form ID, so
```js
trackingPrefix: '22-4142-'
```

This is what the api will use to look up saved forms and the like, if I recall correctly.

Reply:
Thanks! I changed it to 21-4142...

Reply:
Tracking prefix and formid have been changed to '21-4142'